### DISCUSSION TITLE
Is it correct that dispatching events will only work with events names that don't have uppercases?

### DISCUSSION THREAD

**Discussion by ralphjsmit** [alpinejs/alpine#2308] [7. Bugs (archived)] [answered]
I've been working with Alpine's magic $dispatch property lately. It's really great how easy to use it it.

One thing that I noticed (after almost going crazy) is that events like `$dispatch('scrolledIntoView')` and `@scrolledIntoView.window=""` will not work, whilst expressions like `$dispatch('scrolled-into-view')` and `@scrolled-into-view=""` do work.

This made me wonder whether it's indeed the use of capitals here that does it. Is that true?

If it's true, perhaps it would be an idea to add this to the docs, so that other Laravel developers used to using camelCase don't have to figure this out 😁

**Comment by SimoTod** [comment, marked as answer]
It's not true. Dispatch can dispatch any event. Browser attributes are case insensitive and always considered lowercase (it's also in the docs) so you can't write @customEvent.
You probably want to have a look at the camel modifier in the docs as well.

**Comment by ralphjsmit** [comment]
Ah, I totally missed the `.camel` modifier, thanks Simone!
